modules/sdk-coin-avaxc/src/avaxc.ts

```diff
@@ -748,7 +748,10 @@ export class AvaxC extends BaseCoin {
     const tx = await txBuilder.build();
     const payload = tx.signablePayload;
     const signatures = tx.signature.map((s) => Buffer.from(s, 'hex'));
-    const recoverPubky = signatures.map((s) => AvaxpLib.Utils.recoverySignature(_.get(tx, '_network'), payload, s));
+    const network = _.get(tx, '_network');
+    const recoverPubky = signatures.map((s) =>
+      AvaxpLib.Utils.recoverySignature(network as unknown as AvalancheNetwork, payload, s)
+    );
```

Comment:
Weird that I had to make this change 🤔  but talked through with the avaxc team for this.